contracts/cw1-whitelist/src/contract.rs

```diff
@@ -0,0 +1,171 @@
+use crate::error::ContractError;
+use cosmwasm_std::{Addr, DepsMut, Empty, Env, MessageInfo, Response};
+
+use cw1::Cw1;
+use cw1::*;
+use cw_storage_plus::Map;
+use sylvia::contract;
+
+pub struct Cw1WhitelistContract {
+    members: Map<'static, Addr, Empty>,
```

Comment:
Actually, by biggest nit is use by value not reference. Please use:

`members: Map<'static, &Addr, Empty>,`

(And I think you can skip `'static` maybe? if it compiles without)